tt_metal/impl/debug/watcher_server.cpp

```diff
@@ -97,8 +97,9 @@ static void dump_l1_status(FILE *f, Device *device, CoreCoord core, const launch
     data = tt::llrt::read_hex_vec_from_core(device->id(), core, MEM_L1_BASE, sizeof(uint32_t));
     // XXXX TODO(pgk): get this const from llrt (jump to fw insn)
     if (data[0] != 0x2010006f) {
-        log_running_kernels(launch_msg);
-        TT_THROW("Watcher found corruption at L1[0] on core {}: read {}", core.str(), data[0]);
+        // TODO: Change this back to TT_ASSERT when #6159 is fixed.
+        // log_running_kernels(launch_msg);
+        log_warning("Watcher found corruption at L1[0] on core {}: read {}", core.str(), data[0]);
```

Comment:
Hmm, not sure this is the right thing to do.  L1 corruption is a bug and we don't want to miss new regressions

Reply:
We can wait to merge the next couple watcher PRs until #6159 is fixed, so long as someone from the runtime team will look into it soon